pkg/globalconfig/global_config.go

```diff
@@ -209,7 +209,9 @@ func ValidateGlobalConfig() error {
 	}
 
 	if !types.IsValidRouterType(DdevGlobalConfig.Router) {
-		return fmt.Errorf("invalid router: %s, valid router types are %v", DdevGlobalConfig.Router, types.GetValidRouterTypes())
+		output.UserOut.Printf("\nThe only valid router type is %s, but you have router: %s in your global configuration, using %s instead.\n", types.RouterTypeTraefik, DdevGlobalConfig.Router, types.RouterTypeTraefik)
+		DdevGlobalConfig.Router = types.RouterTypeTraefik
+		return nil
```

Comment:
Return was added here to show a warning for the user each time, but when I tested it with:

```yaml
router: nginx-proxy
xdebug_ide_location: "blablabla"
```

DDEV didn't complain about wrong `xdebug_ide_location`.

I don't think it's worth skipping all the further validation checks here.

Reply:
Fixed, thanks for catching the erroneous return there.